rclcpp_lifecycle/include/rclcpp_lifecycle/lifecycle_node.hpp

```diff
@@ -183,7 +183,9 @@ class LifecycleNode : public node_interfaces::LifecycleNodeInterface,
    */
   RCLCPP_LIFECYCLE_PUBLIC
   rclcpp::CallbackGroup::SharedPtr
-  create_callback_group(rclcpp::CallbackGroupType group_type);
+  create_callback_group(
+    rclcpp::CallbackGroupType group_type,
+    bool automatically_add_executor_with_node = true);
```

Comment:
```suggestion
    bool automatically_add_to_executor_with_node = true);
```

how about using the same name with
https://github.com/ros2/rclcpp/blob/3fa511a8a03682eb5e876c0def204b79bd3491c4/rclcpp/include/rclcpp/node.hpp#L146

Reply:
You are right its a typo.